### DISCUSSION TITLE
new release?

### DISCUSSION THREAD

**Discussion by ilf** [transmission/transmission#3803] [Q&A] [answered]
I would love to see a new release.

Since [the 3.00 release](https://github.com/transmission/transmission/releases/tag/3.00), there have been 1.483 commits. There are many useful features and fixes.

How about a 3.01? This would greatly benefit a lot of users.

Also, [the next Debian release freeze](https://release.debian.org/testing/freeze_policy.html) is coming closer, and it would be super-awesome to have a current transmission package in there.

[Release early, release often.](https://en.wikipedia.org/wiki/Release_early,_release_often)

Thanks for all work! <3

**Comment by nevack** [comment]
https://github.com/transmission/transmission/discussions/2287

**Reply by Pentaphon** [reply]
They are way beyond 3.01 now but you probably won't get 4.0.0 in Debian by the time the stable version is out.

**Comment by ilf** [comment]
The freeze for Debian 12 bookworm started: https://lists.debian.org/debian-devel-announce/2023/01/msg00004.html

It would be super-awesome to get Transmission 4.0 into this release.

Thanks for your work. <3

**Reply by ckerr** [reply]
@ilf thanks for the heads-up.

I can't make any promises, but FWIW the feedback from b3 has been pretty good so I _think_ we are nearing the end of beta

**Reply by Pentaphon** [reply]
> The freeze for Debian 12 bookworm started: https://lists.debian.org/debian-devel-announce/2023/01/msg00004.html

2023-02-12 - Milestone 2 - Soft Freeze

It's gonna be close!

**Reply by ckerr** [reply]
Transmission 4.0.0 has been released. Everyone go tell Debian!

**Comment by ckerr** [comment]
I've set the `4.0.0` milestone for Feb 7, 2023.

**Comment by ckerr** [comment, marked as answer]
locking this thread since 4.0.0 is released.
